content/docs/integrations/fleetdm.mdx

```diff
@@ -0,0 +1,230 @@
+   ```
+
+### Configure certificate query
+
+Fleet agents provide a [certificates](https://fleetdm.com/tables/certificates) query that scans a device’s local key store for installed certificates. This query exposes certificates data, including the client certificate SHA-1 fingerprint installed on an enrolled host.
+
+To configure the certificate query, in the Fleet dashboard:
+
+1. Select **Queries.**
+1. Select **Add query.**
+1. Enter the following SQL command in the query field:
+
+   ```sql
+   SELECT * FROM certificates where ca != 1;
+   ```
+
+1. Select **Save** and enter a **Name**, **Description**, and **Frequency**.
+
+   ![Entering a certificate query SQL command in the Fleet dashboard](./img/fleet/fleet-certificate-query.png)
+
+In the Fleet dashboard, you can review a host's certificate details on the **Queries** page:
+
+1. Select **Queries**.
+1. Select the certificate query.
+1. Find a host to review its certificate data.
+
+   ![Reviewing the client certificate SHA-1 fingerprint in the Fleet dashboard](./img/fleet/fleet-certs-query-dashboard.png)
+
+### Create a Fleet policy
+
+In the Fleet dashboard:
+
+1. Select **Policies**.
+1. Select **Add policy**.
+1. Select a policy template, like **Gatekeeper enabled (macOS).**
+1. Save your policy.
+
+At this point, you can add as many policies as you like. You can also mark a policy as critical by selecting the **Critical** checkbox.
+
+![Creating a Fleet policy in the Fleet dashboard](./img/fleet/fleet-policy.png)
+
+:::note
+
+See the Fleet [policies](https://fleetdm.com/docs/rest-api/rest-api#policies) documentation to learn about how policies work.
+
+:::
+
+### Configure Pomerium Enterprise
+
+Next, add the FleetDM plugin as an external data source in the Enterprise Console.
+
+This step requires you to run the [FleetDM plugin](https://github.com/pomerium/datasource/tree/main) with its Docker image.
+
+:::note
+
+These instructions assume a local testing environment using [Docker Compose](https://docs.docker.com/compose/). Adjust as needed for your deployment environment.
+
+:::
+
+1. Add the FleetDM plugin Docker image to Docker Compose:
+
+   ```yaml
+   services:
+     fleetdm:
+       image: docker.cloudsmith.io/pomerium/datasource/datasource:main
+       command:
+         - fleetdm
+         - --api-url=https://pomerium.cloud.fleetdm.com/
+         - --api-token=<YOUR_FLEETDM_API_TOKEN>
+         - --cert-query-id=<YOUR_FLEETDM_CERT_QUERY_ID>
+         - --address=:8080
+       container_name: fleetdm
+       restart: always
+       expose:
+         - 8080
+   ```
+
+1. Run the container:
+
+   ```bash
+   docker compose up
+   ```
+
+In the Enterprise Console:
+
+1. Select **External Data**.
+1. Select **ADD EXTERNAL DATA SOURCE**.
+1. In the **URL** field, enter the URL to the external data source.
+1. Select **SAVE EXTERNAL DATA SOURCE**.
+
+### Create a policy in Pomerium
```

Comment:
we don't seem to explain that the 

- user has to import the CA `mkcert --CAROOT`/rootCA.pem 
- user has to create a new route they want to protect
- user has to select Downstream CA and the uploaded certificate